Content.Server/Forensics/Components/FiberComponent.cs

```diff
@@ -12,5 +12,8 @@ public sealed partial class FiberComponent : Component
 
         [DataField]
         public string? FiberColor;
+
+        [DataField]
+        public string? Fiberprint; // DeltaV, unique glove fibers
```

Comment:
```suggestion
        public string? Fiberprint;
```
@WarMechanic is here too so I'm going to hope and cope it's fine to take it into base game